### DISCUSSION TITLE
Adding Extra Model

### DISCUSSION THREAD

**Discussion by ThesiiNCey** [nextcloud/recognize#988] [Q&A] [answered]
Hey Community :D

is it possible to add this model -> https://github.com/GantMan/nsfw_model

**Comment by marcelklehr** [comment, marked as answer]
Hey @ThesiiNCey 

in principle this is possible. Note, however, that we are currently maintaining this app on a limited effort basis. This means Nextcloud GmbH will currently not invest further development resources ourselves in advancing this app with new features. That doesn't mean there will be no new features, however: We do review and enthusiastically welcome community Pull Requests. Sometimes I work on fun features in my free time. We would be more than excited if you would like to collaborate with us on this. Feel free to reach out here in the comments if you would like to work on this and have questions about how to go about it, how to proceed or would like a short introduction call into the code base. I'm here to help with your questions ✌️ (See https://github.com/nextcloud/recognize/discussions/779 for more information on this)

**Comment by kikislater** [comment]
It could be nice to add a custom pre trained model
